platform/wab/src/wab/client/components/grouping/VirtualTree.tsx

```diff
@@ -13,11 +13,13 @@ export interface RenderElementProps<T> {
     isOpen: boolean;
     toggleExpand: () => void;
   };
+  isSelected?: boolean;
```

Comment:
It felt messy to add top level state here, but seemed wrong to include in `treeState`

Reply:
I think it could be added to treeState as `selectedIndex`, but it is also fine to keep it like this

Reply:
Sure, I can do it either way. Do you think it's better to have `selectedIndex` in the treeState, vs `isSelected`?